app/descriptor/views.py

```diff
@@ -93,15 +93,15 @@ def edit_name(desc_id):
     old_name = descriptor.name
     form = EditDescriptorNameForm()
     if form.validate_on_submit():
-        descriptor.name = form.name.data
         if Descriptor.query.filter(Descriptor.name == form.name.data).first() \
```

Comment:
right now we also show error if we change the name to be the same as it was before, so if we were in the 'Gender' descriptor and we try just changing name to 'Gender' (basically leaving it unchanged), we still show the error that the descriptor 'Gender' already exists. Should we handle this differently by either showing a warning that there is no change or just allow the change?